addons/binding/org.openhab.binding.gpio/src/main/java/org/openhab/binding/gpio/handler/GPIODigitalOutputHandler.java

```diff
@@ -0,0 +1,111 @@
+
```

Comment:
It seems that this file has a missing, outdated or otherwise non standard copyright header, can you fix all of them by running 

`mvn license:format` 

Please commit only the ones for your binding.

Reply:
Let's track the license issues here: [ https://github.com/openhab/openhab2-addons/pull/1334#discussion_r118779994]( https://github.com/openhab/openhab2-addons/pull/1334#discussion_r118779994)